unlock-app/src/components/interface/GlobalErrorConsumer.tsx

```diff
@@ -18,7 +17,7 @@ export function GlobalErrorConsumer({ children, error }: Props) {
   // the error object in this case is coming from the redux store and has a structure of {name, data}
   if (error) {
     const Error = mapErrorToComponent(error)
-    return <Layout title="">{Error}</Layout>
```

Comment:
the consumer is inside Layout now, so it is unnecessary to render a Layout inside the consumer